rook/helmfile.d/charts/rook-ceph-rules/files/rules.yaml

```diff
@@ -679,29 +679,7 @@ groups:
           description: >
             Root volume is dangerously full: {{ $value | humanize }}% free.
 
-      # alert on packet errors and drop rate
```

Comment:
Maybe add a comment saying that this alert has been removed. Fairly certain we have written a similar note somewhere in this file already.